lambda-deployment-orca/src/main/java/com/amazon/aws/spinnaker/plugin/lambda/LambdaDeploymentStage.java

```diff
@@ -43,8 +43,8 @@ public LambdaDeploymentStage() {
     public void taskGraph(@Nonnull StageExecution stage, @Nonnull TaskNode.Builder builder) {
         logger.debug("taskGraph for Aws.LambdaDeploymentStage");
         builder.withTask("lambdaCreateTask", LambdaCreateTask.class);
-        builder.withTask("lambdaUpdateCodeTask", LambdaUpdateCodeTask.class);
         builder.withTask("lambdaUpdateConfigTask", LambdaUpdateConfigurationTask.class);
+        builder.withTask("lambdaUpdateCodeTask", LambdaUpdateCodeTask.class);
```

Comment:
There should be a verification task between the update config and update code. This decouples those calls a bit to prevent the concurrent operation failures
```suggestion
        builder.withTask("lambdaVerificationTask", LambdaVerificationTask.class);
        builder.withTask("lambdaUpdateCodeTask", LambdaUpdateCodeTask.class);
```